docs/detections/api/rules/rules-api-bulk-actions.asciidoc

```diff
@@ -12,6 +12,8 @@ NOTE: Console supports only {es} APIs and doesn't allow interactions with {kib}
 [[bulk-actions-rules-api-create]]
 ==== Bulk create
 
+IMPORTANT: This API has been deprecated since the 8.2 release, and its end of life is expected in Q4 2023. Please use the <<bulk-actions-rules-api-action, bulk action API>> as an alternative.
```

Comment:
```suggestion
IMPORTANT: This API has been deprecated since version 8.2, and is scheduled for end of life in Q4 2023. Please use the <<bulk-actions-rules-api-action, bulk action API>> instead.
```
Curious to see the other reviewers' thoughts on these suggestions.

Reply:
Thanks for the suggestion! Applied it